cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ForeachTest.scala

```diff
@@ -28,36 +28,28 @@ class ForeachTest extends DocumentingTest with QueryStatisticsTestSupport {
-                  |       (b:Person {name: 'B'}),
-                  |       (c:Person {name: 'C'}),
-                  |       (d:Person {name: 'D'}),
-                  |       (a)-[:KNOWS]->(b),
-                  |       (b)-[:KNOWS]->(c),
-                  |       (c)-[:KNOWS]->(d)""")
-    synopsis("The `FOREACH` clause is used to update data within a list, whether components of a path, or result of aggregation.")
-    p(
-      """* <<query-foreach-introduction, Introduction>>
-        |* <<foreach-mark-all-nodes-along-a-path, Mark all nodes along a path>>
-      """.stripMargin)
+                  #(b:Person {name: 'B'}),
+                  #(c:Person {name: 'C'}),
+                  #(d:Person {name: 'D'}),
+                  #(a)-[:KNOWS]->(b),
+                  #(b)-[:KNOWS]->(c),
+                  #(c)-[:KNOWS]->(d)""".stripMargin('#'))
+    synopsis("The `FOREACH` clause is used to update data within a collection whether components of a path, or result of aggregation.")
     section("Introduction", "query-foreach-introduction") {
-      p(
-        """Lists and paths are key concepts in Cypher.
-          |`FOREACH` can be used to update data, such as executing update commands on elements in a path, or on a list created by aggregation.""".stripMargin)
+      p("""Lists and paths are key concepts in Cypher.
+          #The `FOREACH` clause can be used to update data, such as executing update commands on elements in a path, or on a list created by aggregation.""".stripMargin('#'))
       p("""The variable context within the `FOREACH` parenthesis is separate from the one outside it.
-          |This means that if you `CREATE` a node variable within a `FOREACH`, you will _not_ be able to use it outside of the foreach statement, unless you match to find it.""".stripMargin)
-      p(
-        """Within the `FOREACH` parentheses, you can do any of the updating commands -- `CREATE`, `MERGE`, `DELETE`, and `FOREACH`.""".stripMargin)
-      p("""If you want to execute an additional `MATCH` for each element in a list then `UNWIND` (see <<query-unwind>>) would be a more appropriate command.""".stripMargin)
+          #This means that if you `CREATE` a node variable within a `FOREACH`, you will _not_ be able to use it outside of the foreach statement, unless you match to find it.""".stripMargin('#'))
+      p("Within the `FOREACH` parentheses, you can do any of the updating commands -- `SET`, `REMOVE`, `CREATE`, `MERGE`, `DELETE`, and `FOREACH`.")
+      p("If you want to execute an additional `MATCH` for each element in a list then the <<query-unwind,`UNWIND`>> clause would be a more appropriate command.")
       graphViz()
     }
     section("Mark all nodes along a path", "foreach-mark-all-nodes-along-a-path") {
-      p(
-        """This query will set the property `marked` to true on all nodes along a path.""".stripMargin)
+      p("This query will set the property `marked` to true on all nodes along a path.")
```

Comment:
```suggestion
      p("This query will set the property `marked` to `TRUE` on all nodes along a path.")
```

Reply:
Write boolean literals (true and false) in lower case. - https://neo4j.com/docs/cypher-manual/4.2/styleguide/

Reply:
so i will fix it to be `true`